theforeman.org/yaml/views/release.yml

```diff
@@ -0,0 +1,21 @@
+- view-template:
+    name: 'Foreman {major}.{minor}'
+    description: 'Foreman {major}.{minor} jobs'
+    view-type: list
+    regex: '.*{major}[_\.]{minor}.*'
+
+- project:
+    name: 'foreman-releases'
+    views:
+        - '{name}-template-{major}.{minor}'
```

Comment:
I think this needs to match the `name:` defined on https://github.com/theforeman/jenkins-jobs/pull/176/files#diff-b6bd4b212bdd5eae7e1f14909bf3b48d97db0a2b5d14afd613c56b60ddc54528R2 in order for it to link them up properly